coursedata/level-defaults/es.yaml

```diff
@@ -117,8 +117,8 @@
       -
         name: "repeat"
         explanation: "`repeat` e `if` combinados"
-        example: "Ejemplo: if color is verde repeat 3 times print '¡Qué bonito!' else repeat 5 times print 'A mi me gusta otro'"
-        demo_code: "color is ask ¿Cuál es tu color favorito?\nif color is verde repeat 3 times print '¡Qué bonito!' else repeat 5 times print 'A mi me gusta otro'"
+        example: "Ejemplo: if producto is 50 repeat 3 times print 'correcto!' else repeat 5 times print 'incorrecto!'"
+        demo_code: "producto is ask Cuanto es 10 veces 5?\nif producto is 50 repeat 3 times print 'correcto!' else repeat 5 times print 'incorrecto!'"
```

Comment:
this should probably be `10 por 5` if you mean "ten times five" (multiplication).

Reply:
Hi @lpmi-13 ! Native Spanish speaker here :) Both "por" and "veces" are used in school settings for multiplication, but "veces" seems to be more prevalent than "por" in educational settings, since it's more intuitive - usually adults use more "por" because it's shorter. I think it's best to leave it as is, but I appreciate the review!

Reply:
oh really? I apologize, I had no idea! Thanks for the thorough explanation.

Reply:
...this is probably an appropriate place for an "estoy embarazado" joke

Reply:
Hahaha, no worries! In fact, your question made me look deeper into this, to make sure that "veces" was the right word - I had to look it up myself, because I didn't really think about it when I wrote the translation!